lms/static/certificates/sass/_layouts.scss

```diff
@@ -0,0 +1,324 @@
+        .accomplishment-recipient {
+            width: span(12);
+
+            @include susy-breakpoint($bp-large, $susy) {
+                width: span(10);
+            }
+        }
+
+        .accomplishment-summary,
+        .accomplishment-statement-detail {
+           width: span(12);
+
+           @include susy-breakpoint($bp-large, $susy) {
+               width: span(8);
+           }
+        }
+    }
+
+    .accomplishment-orgs {
+        margin-bottom: 0;
+
+        .wrapper-orgs {
+            text-align: center;
+        }
+
+        .wrapper-organization {
+            display: inline-block;
+            vertical-align: middle;
+            width: span(6);
+            height: rem(100);
+            margin-right: spacing-horizontal(base);
+
+            &:last-child {
+                margin-right: 0;
+            }
+
+            @include susy-breakpoint($bp-medium, $susy) {
+                width: span(3);
+            }
+
+            @include susy-breakpoint($bp-large, $susy) {
+                width: span(2);
+            }
+
+            @include susy-breakpoint($bp-huge, $susy) {
+                width: span(2);
+            }
+        }
+
+        .organization {
+            @include size(100%);
+            position: relative;
+            margin-bottom: spacing-vertical(small);
+        }
+
+        .organization-logo {
+            max-width: 80%;
+            position: absolute;
+            top: 50%;
+            left: 50%;
+            transform: translate(-50%, -50%);
+        }
+    }
+
+    .accomplishment-signatories {
+
+        .wrapper-signatories {
+            text-align: center;
+        }
+
+        .signatory {
+            display: inline-block;
+            vertical-align: middle;
+            width: span(12);
+
+            @include susy-breakpoint($bp-medium, $susy) {
+                width: span(4);
+            }
+
+            @include susy-breakpoint($bp-large, $susy) {
+                width: span(3);
+            }
+
+            @include susy-breakpoint($bp-huge, $susy) {
+                width: span(3);
+            }
+
+            .signatory-signature {
+                display: block;
+                max-width: 100%;
+                height: rem(100);
+                padding: spacing-vertical(small) spacing-horizontal(small);
+            }
+        }
+    }
+
+    .wrapper-accomplishment-metadata {
+        @extend %layout-wrapper;
+
+        .accomplishment-metadata {
+            @extend %layout;
+        }
+
+        .wrapper-metadata .metadata {
+            @extend %divider-2;
+            margin-bottom: spacing-vertical(small);
+            border-bottom-color: palette(grayscale, x-light);
+            padding-bottom: spacing-vertical(small);
+
+            &:last-child {
+                border-bottom: none;
+                padding-bottom: 0;
+            }
+
+            @include susy-breakpoint($bp-medium, $susy) {
+                @include gallery(4);
```

Comment:
NOTE: new susy grid-management syntax - this creates a column layout with row breaks set for nth elements based on what's passed in (in this case, every 4 elements). See - http://susydocs.oddbird.net/en/latest/toolkit/#gallery